gst/tensor_decoder/tensordec-boundingbox.c

```diff
@@ -730,11 +730,11 @@ draw (GstMapInfo * out_info, bounding_boxes * bdata, GArray * results)
     detectedObject *a = &g_array_index (results, detectedObject, i);
 
     /* 1. Draw Boxes */
-    x1 = MAX (0, (bdata->width * a->x) / bdata->i_width);
+    x1 = (bdata->width * a->x) / bdata->i_width;
     x2 = MIN (bdata->width - 1,
         (bdata->width * (a->x + a->width)) / bdata->i_width);
 
-    y1 = MAX (0, (bdata->height * a->y) / bdata->i_height);
+    y1 = (bdata->height * a->y) / bdata->i_height;
```

Comment:
We need to keep ```MAX```. We sometimes get negative values, which breaks (seg faults) memory.

Reply:
current code, x and y in struct detectedObject cannot be set negative value.
line 594
        result->x = MAX (0, x); \
        result->y = MAX (0, y); \

Reply:
Ah.. got it.